docs/advanced_usage.rst

```diff
@@ -306,10 +306,21 @@ Before upgrading, we recommend resolving any deprecation warnings raised in your
 
 .. _upgrade-0.x:
 
-0.x Release Line
-^^^^^^^^^^^^^^^^
+Upgrading from 0.x to 1.x
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As of v0.60.0, you can enable warning filters for ddtrace library deprecations.
+
+To identify the deprecated environment variables, search a code base for the following patterns::
+
+    git grep -e "DATADOG_[A-Z_]*" \
+      -e "DATADOG_SERVICE_NAME" \
+      -e "DD_CALL_BASIC_CONFIG" \
+      -e "DD_LOGGING_RATE_LIMIT" \
+      -e "DD_SERVICE_NAME" \
+      -e "DD_TRACER_PARTIAL_FLUSH_ENABLED" \
+      -e "DD_TRACER_PARTIAL_FLUSH_MIN_SPANS"
```

Comment:
Should this come after the pytest/warnings stuff? or before L312?

Basically it reads as: "You can not enable warnings, here is how to grep for env variable usage"

Reply:
Fair point